client/src/js/services.js

```diff
@@ -57,11 +58,12 @@ module.exports = function (app) {
          */
         getItems: function () {
           //return the promise directly.
-          return $http.get(WS_SERVER + '/api/vizabi/gapminder_tools/related_items/')
+          return Promise.resolve(require('../config/related-items.json'))
```

Comment:
you don't need bluebird, there are `$q` service from promises
+ we try to avoid using promises